src/shared/UIElements/FormItem/FormItem.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+    <b-input
+        v-if="type == InputType.Text || type == InputType.Number"
+        :id="inputId"
+        v-model="inputValue"
+        :type="(type == InputType.Number) ? 'number' : 'text'"
+        :class="inputClass"
+        v-bind="inputAttributes"
+    ></b-input>
+    <b-textarea
```

Comment:
This is just a minor formatting issue, but i think would look better if there was empty lines after this considering the next line is a different type of input.